shared/fs/index.stories.js

```diff
@@ -0,0 +1,24 @@
+// @flow
```

Comment:
Is this the file we'd play with for StoryBook?

Also, I noticed that you were running that as a server on your localhost. How do we access it?